python/tests/types/test_vision.py

```diff
@@ -264,6 +261,27 @@ def test_draw_texts():
     assert np.array_equal(pil_image.to_numpy(), expected)
 
 
+def test_draw_box_with_label():
+    data = np.random.randint(0, 255, size=(100, 100, 3), dtype=np.uint8)
+    img = Image.from_array(data)
+
+    box1 = Box2d(1, 2, 10, 12)
+    box2 = Box2d(20, 20, 40, 40)
+    draw_boxes = img | (box1 + "label1") | box2.with_label("label2")
```

Comment:
this pr adds new operator `+` to express

align with the base drawable first and then append the layer to the base drawable

Reply:
it is an important design and I think you should describe about it in the pull request desc part.

Reply:
well, there is a detailed desc in #550

Reply:
commented in #550, for Mask, the `+` operator is also needed.

Reply:
Currently,  `+` is only an alias of `.with_label`, and it's only a convenient feature between `box2d` and `label`. If we should expand it to more classes, maybe we should design a coherent semantic of syntax sugar for all classes.